### ISSUE TITLE
Gateway metrics and statistics

### ISSUE THREAD

**Issue by okjodom** [fedimint/fedimint#2624]
Gateway operators would benefit from historical stats on the services offered to federations and it's users. These may include
- **volume**: count of all payments processed
- **size**: aggregate (in sats) of all payments processed
- **outcome**: rates of success / failure in processing payments
- **derived**: derived stats on individual payments processed, for example, the average size of a payment from a federation

**Note**: metrics might be aggregated along these dimensions
- incoming vs outgoing paymens
- per federation / globally for this gateway
- by time ranges

**Comment by nassersaazi** [comment]
seems related to #1583 .wonder if you have some time for a chat about implementation details sometime this week @okjodom

**Comment by elsirion** [comment]
@m1sterc001guy can this be closed after your work in 0.6/0.7?

**Comment by m1sterc001guy** [comment]
Yes all of this is possible to compute with the `payment-summary` and `payment-log` APIs
